build.gradle

```diff
@@ -110,6 +112,12 @@ subprojects {
     libraries = new Properties()
     file(new File(rootDir, "dependencies.properties")).withReader{ libraries.load((Reader) it) }
 
+    def epoll_suffix = "";
+    if (osdetector.classifier in ["linux-x86_64"]) {
+      // The native code is only pre-compiled on certain platforms.
+      epoll_suffix = ":" + osdetector.classifier
+    }
```

Comment:
I'm not a gradle expert, but won't this only change when gax-java builds? It won't be decided by the end-user's environment.

Reply:
You are correct. I have confirmed with the grpc-go team, epoll is already shaded as part of as part of grpc-netty-shaded. So we don't even need to specify the dependency here. Thanks for pointing that out!